Goobi/src/org/goobi/production/flow/statistics/enums/TimeUnit.java

```diff
@@ -239,6 +239,8 @@ private DateTimeFormatter getFormatter(){
 		case quarters:
 			// has to be extended by the calling function
 			return DateTimeFormat.forPattern("yyyy");
+		case simpleSum:
+			break;
```

Comment:
Constant simpleSum could even removed as this constant is never used outside this class.